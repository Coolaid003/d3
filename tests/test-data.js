require("./../lib/env-js/envjs/node");
require("./../lib/sizzle/sizzle");
require("./../d3");

var vis = d3.select("body").selectAll("span")
    .data(d3.range(10));

vis.enter().append("span")
    .text(String);

console.log("data bind:");
console.log("  ", document.body.innerHTML);
console.log("");

console.log("data retrieve (old selection):");
console.log("  ", vis.data().join(","));
console.log("");

console.log("data retrieve (reselection):");
console.log("  ", d3.selectAll("span").data().join(","));
console.log("");
