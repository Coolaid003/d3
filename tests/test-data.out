data bind:
   <span>0</span><span>1</span><span>2</span><span>3</span><span>4</span><span>5</span><span>6</span><span>7</span><span>8</span><span>9</span>

data retrieve (old selection):
   

data retrieve (reselection):
   0,1,2,3,4,5,6,7,8,9

